stats/opentelemetry/opentelemetry.go

```diff
@@ -37,20 +37,20 @@ var canonicalString = internal.CanonicalString.(func(codes.Code) string)
 
 var joinDialOptions = internal.JoinDialOptions.(func(...grpc.DialOption) grpc.DialOption)
 
-// MetricName is a name of a metric.
-type MetricName string
+// Metric is an identifier for a metric provided by this package.
+type Metric string
 
 // Metrics is a set of metrics to record. Once created, Metrics is immutable,
 // however Add and Remove can make copies with specific metrics added or
 // removed, respectively.
 type Metrics struct {
 	// metrics are the set of metrics to initialize.
-	metrics map[MetricName]bool
+	metrics map[Metric]bool
 }
 
-// NewMetrics returns a Metrics with the metrics provided specified.
-func NewMetrics(metrics ...MetricName) *Metrics {
-	newMetrics := make(map[MetricName]bool)
+// NewMetrics returns a Metrics containing the Metric's provided.
```

Comment:
Another choice is to say "containing metrics." - directly referencing the parameter's name.

Reply:
Good point, switched.